### ISSUE TITLE
When is the release or Redis Stack 7.0 Stable?

### ISSUE THREAD

**Issue by yorkfread** [redis-stack/redis-stack#438]
In my quest for employing Redis Stack across all servers, with an ardent desire to embrace the 7.0 iteration of Redis, I find myself longing for the advent of its stable release. Pray enlighten me as to the approximate timeline for its forthcoming publication? Thx.

**Comment by chayim** [comment]
Such a desire, I too share yon @yorkfread. Betwixt the joy of releases I must share that yon release, 7.0 is only be a series of explorations. However, I share with thee the joy of our beloved 7.2 series, a soon to be final release, one which thou canst soon deploy upon thy servers.

tl;dr 7.2 is coming soon - give it a go. The final version is coming shortly!

**Comment by MichaelDeutschCoding** [comment]
@chayim 
Much of a full orbit of yon moon's chariot has completed its course. Wouldst thou be so kind as to impart unto us the latest dispatch regarding the anticipated release date of the forthcoming software rendition? My team doth hold misgivings about the use of release-candidate versions. Your expeditious communication would be embraced with sincere thankfulness.

tl;dr: when?

**Comment by chayim** [comment]
Alas @MichaelDeutschCoding  I must beseech thee for they patience, and your servant's failure. With narry a second thought, 'twas released, lo a fortnight prior. Thus with tears in mine eyes, I apologize for missing thy fair message in the flood that is mine notifications.

tl;dr [already](https://github.com/redis-stack/redis-stack/releases/tag/v7.2.0-v0)!
